examples/custom-server-express/server.js

```diff
@@ -9,15 +8,6 @@ app.prepare()
 .then(() => {
   const server = express()
 
-  server.use((req, res, next) => {
-    const accept = accepts(req)
-    const type = accept.type(['json', 'html'])
-    if (type === 'json') {
-      return handle(req, res)
-    }
-    next()
-  })
```

Comment:
I think this is something pretty great.
I'll change the API to be like this.

Anyway, how about changing it back to `.json`. Accept headers won't give us any advantages but problems.

With this way, we could keep actual URLs and page JSON separately. Actually, that's how they work.